nextroute/schema/output_basic.go

```diff
@@ -18,7 +18,6 @@ type VehicleOutput struct {
 	ID            string       `json:"id"`
 	Route         []StopOutput `json:"route"`
 	RouteDuration int          `json:"route_duration"`
-	RouteDistance int          `json:"route_distance"`
```

Comment:
I believe that was added already. The necessary distanceExpression came in with this PR https://github.com/nextmv-io/platform/pull/296

Reply:
If you do not use that constraint (for example if it is disabled via options or it is not specified in the input) then that expression is not added anywhere. @davidrijsman and I discussed this morning and we have to find a good way to report distance even if it is not used in constraints or objectives. There is a ticket for that already: [ENG-2800](https://linear.app/nextmv/issue/ENG-2800/add-distance-reporting-to-the-output-in-the-basic-formatter)

Reply:
Also, the API currently does not have any method that allows you to get the cumulative distance for a vehicle or vehicle type.

Reply:
Sounds good, thanks for clarification